include/eve/arch/cpu/wide.hpp

```diff
@@ -869,41 +885,41 @@ inline namespace EVE_ABI_NAMESPACE
     // Logical operations
     //==============================================================================================
     //! @brief Element-wise equality comparison of two eve::wide
-    friend EVE_FORCEINLINE logical<wide> operator==(wide v, wide w) noexcept
+    friend  EVE_FORCEINLINE auto operator==(wide const& v, wide const& w) noexcept
```

Comment:
Oh come on, we can pass by value. But w/e.

Reply:
this is a remnant of an old test, will clean up